### PR TITLE
Add new on-campus landing page

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-3538](https://trello.com/c/JuTvyvH8/3538-create-new-on-campus-landing-page)

### Context

The Direct Marketing team want to try a new on-campus landing page; Havas have came up with a design which we have implemented with a view to reusing the new components.

### Changes proposed in this pull request

- Add PhotoQuoteListComponent for on-campus landing page

The new on-campus landing page designed by Havas has a list of quotes and associated photographs.

Add component to render a list of quotes with photos; the layout is fairly complex so I've split the grid layout in the CSS to make it easier to read.

- Add food for thought page

This is the new on-campus landing page. It uses the new `PhotoQuoteListcomponent` to display a number of quotes. The markdown scss had to be updated to allow for a yellow box-style heading on this page.

- Make subject specific header more generic

We want to use the subject specific header/hero in other areas of the website.

Refactor it to be a generic campaign header that can be used on other pages.

- Update on-campus hero section

Use the generic campaign header with an updated cake image.

### Guidance to review

The page can be previewed [here](https://review-get-into-teaching-app-2765.london.cloudapps.digital/landing/food-for-thought).

### Checklist of things still to do

- [x] Noindex the page
- [x] Consider adding a folder structure eg /landing
- [x] Add a promo module
- [x] Add intro paragraph and heading from Havas
- [x] Add case study copy and images
- [x] Review Havas copy and update if needed
- [ ] Update promo module link (and photo) prior to merging

### CODE REVIEW

File: app/webpacker/styles/components/photo-quote-list.scss [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      picture img {
+        width: 100%;
+        max-height: 250px;
+        object-fit: cover;
+        object-position: top;
+      }
+
+      h3 {
+        max-width: 100%;
+        text-align: right;
```
Comment (gemmadallmandfe) [inline app/webpacker/styles/components/photo-quote-list.scss:120]: Is there a reason to right-align the h3s @ethax-ross @gazcarr ? 

I'd rather left-align them from an accessibility perspective if possible - plus I'm not sure where else we right-align anything?

Comment (gemmadallmandfe) [reply app/webpacker/styles/components/photo-quote-list.scss:120]: Just realised the Havas design right-aligned on mobile, but left-aligned on desktop

Can we left-align on both? Any brand issues with this @gazcarr ?

**Comment by gemmadallmandfe** [comment]
Paging @EllieNodder @GDW7 to ask if we should have a (light-touch) CTA at the end of this landing page, possibly to the ETA page?  Promo module? Something else? Or nothing?

**Comment by EllieNodder** [comment]
> Paging @EllieNodder @GDW7 to ask if we should have a (light-touch) CTA at the end of this landing page, possibly to the ETA page? Promo module? Something else? Or nothing?

Seems like the perfect page to me for adding an ETA promo - exactly the right audience

**Comment by gemmadallmandfe** [comment]
> > Paging @EllieNodder @GDW7 to ask if we should have a (light-touch) CTA at the end of this landing page, possibly to the ETA page? Promo module? Something else? Or nothing?
> 
> Seems like the perfect page to me for adding an ETA promo - exactly the right audience

Would you be able to add this (next week) @EllieNodder ?

**Comment by gemmadallmandfe** [comment]
Thanks for adding the promo @EllieNodder! That looks fine - if we get a proper ETA image, it would probably be worth updating it here (or otherwise shortening the 'Explore teaching advisers' title so the person's eyes aren't covered on mobile?). Page is not due to go into production until end of next week though so let's leave for now

**Comment by EllieNodder** [comment]
@ethax-ross have added an ETA promo module - it seems to be coming through in the review app, just not sure I've done it in the _right_ way. The backend of the page looks diff to elsewhere where we have the promo modules so have literally just copied the whole promo rather than the usual 'promo_content:' thing in the frontmatter. Does it look okay to you?

**Comment by EllieNodder** [comment]
> Thanks for adding the promo @EllieNodder! That looks fine - if we get a proper ETA image, it would probably be worth updating it here (or otherwise shortening the 'Explore teaching advisers' title so the person's eyes aren't covered on mobile?). Page is not due to go into production until end of next week though so let's leave for now

Yep, am chasing Steph/Andy about the image

File: app/views/content/food-for-thought/_quotes.html.erb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<section class="container">
+  <article class="markdown campaign">
+    <h2>Lorem ipsum dolor sit amet, consectetur adipiscing elit</h2>
```
Comment (gemmadallmandfe) [inline app/views/content/food-for-thought/_quotes.html.erb:3]:
```suggestion
    <h2>Make uni life a doddle with our bite-sized hacks</h2>
```

**Comment by GDW7** [comment]
@gemmadallmandfe @EllieNodder @ethax-ross I spotted and corrected a small typo in quote 4. Otherwise it looked good to me.

File: app/webpacker/styles/components/photo-quote-list.scss (hunk 1 above)
Comment (GDW7) [reply app/webpacker/styles/components/photo-quote-list.scss]: @gemmadallmandfe @ethax-ross @gazcarr on desktop the quote components are  looking really quite left aligned (lots of white space to the right). Just wanted to check we're happy with this.

**Review by leoapost — APPROVED** [review]
Looks ok to me

File: app/views/content/food-for-thought/_quotes.html.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        {
+          image: "media/images/content/food-for-thought/Joel_Quilter.jpg",
+          heading: "Set a weekly budget",
+          text: "I tried to have a weekly budget. I’d do my food shopping online, which made it easier to take advantage of deals and stay on top of how much exactly I’d been spending. I also had a notes page on my phone where I’d add in what I spent on nights out.",
+          accreditation: ["Joel Quilter"],
+        },
+        {
+          image: "media/images/content/food-for-thought/Johnoi_Josephs.jpg",
+          heading: "Don’t reinvent the wheel",
+          text: "Be creative in how you learn but don’t reinvent the wheel. Degrees are a different level of intensity, but the framework is the same as A-levels or BTECs. Whatever got you here will continue to work! Consistency and intentionality will trump any overnight success - and that’s the same in whatever stage in life.",
```
Comment (gemmadallmandfe) [inline app/views/content/food-for-thought/_quotes.html.erb:38]:
```suggestion
          text: "Be creative in how you learn but don’t reinvent the wheel. Degrees are a different level of intensity, but the framework is the same as A levels or BTECs. Whatever got you here will continue to work! Consistency and intentionality will trump any overnight success - and that’s the same in whatever stage in life.",
```

**Pull request merged**